datacontract/imports/avro_importer.py

```diff
@@ -45,8 +46,13 @@ def import_avro(data_contract_specification: DataContractSpecification, source:
         data_contract_specification.models = {}
 
     try:
-        with open(source, "r") as file:
-            avro_schema = avro.schema.parse(file.read())
+        if source.startswith("sftp://"):
```

Comment:
I am confused regarding this change, which happens in many formatters. sftp support is primarly for the data, not the metadata. I think we can remove this, or did I miss something?